config/templates/kava/evm/initstate/.kava/config/genesis.json

```diff
@@ -0,0 +1,2454 @@
+          "account_number": "0",
+          "address": "kava1n96qpdfcz2m7y364ewk8srv9zuq6ucwduyjaag",
+          "pub_key": null,
+          "sequence": "0"
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "account_number": "0",
+          "address": "kava1sw54s6gq76acm35ls6m5c0kr93dstgrh6eftld",
+          "pub_key": null,
+          "sequence": "0"
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "account_number": "0",
+          "address": "kava1t4dvu32e309pzhmdn3aqcjlj79h9876plynrfm",
+          "pub_key": null,
+          "sequence": "0"
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "account_number": "0",
+          "address": "kava1wuzhkn2f8nqe2aprnwt3jkjvvr9m7dlkpumtz2",
+          "pub_key": null,
+          "sequence": "0"
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "account_number": "0",
+          "address": "kava173w2zz287s36ewnnkf4mjansnthnnsz7rtrxqc",
+          "pub_key": null,
+          "sequence": "0"
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.ModuleAccount",
+          "base_account": {
+            "account_number": "0",
+            "address": "kava1cj7njkw2g9fqx4e768zc75dp9sks8u9znxrf0w",
+            "pub_key": null,
+            "sequence": "0"
+          },
+          "name": "kavadist",
+          "permissions": [
+            "minter"
+          ]
+        },
+        {
+          "@type": "/cosmos.auth.v1beta1.ModuleAccount",
+          "base_account": {
+            "account_number": "0",
+            "address": "kava1mfru9azs5nua2wxcd4sq64g5nt7nn4n8s2w8cu",
+            "pub_key": null,
+            "sequence": "0"
+          },
+          "name": "swap",
+          "permissions": null
+        },
+        {
+          "@type": "/cosmos.vesting.v1beta1.PeriodicVestingAccount",
+          "base_vesting_account": {
+            "base_account": {
+              "account_number": "0",
+              "address": "kava1fwfwmt6vupf3m9uvpdsuuc4dga8p5dtl4npcqz",
+              "pub_key": null,
+              "sequence": "0"
+            },
+            "delegated_free": [],
+            "delegated_vesting": [],
+            "end_time": "1664632800",
+            "original_vesting": [
+              {
+                "amount": "560159828",
+                "denom": "ukava"
+              }
+            ]
+          },
+          "start_time": "1633096800",
+          "vesting_periods": [
+            {
+              "amount": [
+                {
+                  "amount": "560159828",
+                  "denom": "ukava"
+                }
+              ],
+              "length": "31536000"
+            }
+          ]
+        },
+        {
+          "@type": "/ethermint.types.v1.EthAccount",
+          "base_account": {
+            "address": "kava1ak4pa9z2aty94ze2cs06wsdnkg9hsvfkp40r02",
```

Comment:
node: this is the eth address of the validator key, which we don't have a keyring for. Should we include one for this as well so we can send transactions via evm?

Reply:
Right, I'll just include it